crates/cxx-qt-gen/src/lib.rs

```diff
@@ -63,11 +63,17 @@ mod tests {
         }
     }
 
+    /// Helper for creating cxx_named Names, usually for camelcase cxx names
+    pub fn mock_name_with_cxx(name: &str, cxx: &str) -> Name {
```

Comment:
Move this into `name.rs` as an associated function, like `mock` and `mock_namespaced`.

Reply:
done